src/test/java/uk/gov/hmcts/reform/em/orchestrator/stitching/StitchingServiceTest.java

```diff
@@ -27,6 +41,45 @@
     @Value("${em-rpa-stitching-api.resource}")
     private String stitchingResource;
 
+    @MockBean
+    private JwtDecoder jwtDecoder;
+
+    @Autowired
+    private WebApplicationContext context;
+
+    private static final String ID_TOKEN = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9"
```

Comment:
this is duplicate code everywhere, we should just disable security for unit tests: https://stackoverflow.com/questions/31169720/disable-security-for-unit-tests-with-spring-boot

Reply:
I have moved the duplicate code to BaseTest.java now